### DISCUSSION TITLE
"Route not found" in 3.0.11 with BRouter (Beta)

### DISCUSSION THREAD

**Discussion by WolfgangZ42** [devemux86/cruiser#69] [Q&A] [answered]
It's strange, but I get "Route not found" in the newest version of Cruiser: 
![2023-03-06 22 24 31](https://user-images.githubusercontent.com/121806238/223235963-0f0be4ea-76f7-4b2f-a14b-7871405c9d16.png)
I just set a start and an end - nothing else ...

Importing a navigation gpx in routing is possible, but not changing anything:
![2023-03-06 22 39 42](https://user-images.githubusercontent.com/121806238/223238453-756ffccb-e909-4a0c-a2b3-1abf2fec7738.png)    ![2023-03-06 22 39 48](https://user-images.githubusercontent.com/121806238/223238365-995bd111-3163-4463-bf53-a3b5ea6dc9bb.png)  

I'm running Android 10 (API 29) on Sony H8216 (1080x2160, 480 dpi)

**Comment by airald** [comment]
I am experiencing similar issues, not related just on 3.0.11 version. Some routes that I navigate in previous versions, now appears the message "Route not found". I've tried changing Brouter preferences with no success. Some routes are possible with Valhalla, and some other not.

Sometimes BRouter says that there's a waypoint that is not over a transitable path (in this situation I can use the straight line function) but some others, simply there's no route. 

I've been using Kurviger and Cruiser and never had this kind of problems when routing, just when the waypoint is on a path not transitable by motor vehicles. In this situation, I try to use a "walk profile", but I don't succeeded. 

Maybe my answer should be not here, I'm not sure where's exactly the problem on routing. 

It happens on many devices, different brands and Android versions.

**Comment by devemux86** [comment, marked as answer]
If you are using the BRouter (Beta) version, _you need to update it_ to work with the new Cruiser.

This is mentioned in the [BRouter topic](https://github.com/devemux86/cruiser/discussions/15), where the latest version of BRouter (Beta) is available: https://github.com/devemux86/cruiser/discussions/15#discussion-4757829

---

For anything else, please post a route example and attach the app .pref file, so I can check the report.
("Settings | Application | Preferences | Export" or search settings for "preferences")

Note: routers have different sensitivity for snap the waypoints on road network
and the avoid options can prohibit the calculation, especially with motor profiles.

You can try with disabled avoid options and zoom in enough to place the waypoints near routable network.

**Reply by WolfgangZ42** [reply]
I read "this is compatible to 3.0.11" but didn't see the need to update ...
Now I'm confused - the latest version of the Axel is the 1.6.3n - but I already use the 1.6.4y. What should I do?

**Reply by devemux86** [reply]
> I read "this is compatible to 3.0.11" but didn't see the need to update ...

I improved the description in https://github.com/devemux86/cruiser/discussions/15#discussioncomment-5215132

> the latest version of the Axel is the 1.6.3n - but I already use the 1.6.4y

Axel created the Beta apk with 1.6.3 text, but it's just text, the internal version has increased,
so you can install it on top of your existing BRouter installation.

**Reply by WolfgangZ42** [reply]
Installed - everything is fine! Thanks a lot!

**Reply by devemux86** [reply]
For reference: BRouter (Beta) has changed something internally,
Cruiser 3.0.11 has been changed to support it, so it needs the latest BRouter (Beta) apk. :slightly_smiling_face: 

---

BRouter 1.6.3 release on Google Play is working properly as always.

**Comment by devemux86** [comment]
A new version of BRouter (Beta) is available: https://github.com/devemux86/cruiser/discussions/15#discussion-4757829
